src/neo/Consensus/ConsensusService.cs

```diff
@@ -371,6 +371,10 @@ private void OnRecoveryMessageReceived(ConsensusPayload payload, RecoveryMessage
                         if (ReverifyAndProcessPayload(commitPayload)) validCommits++;
                 }
             }
+            catch (IndexOutOfRangeException e)
+            {
+                Log($"invalid recovery message with illegal validator index. {e}");
```

Comment:
I think that it's better to log any `Exception`

Reply:
done.